packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -1155,5 +1155,10 @@ export const METRICS_OPERATIONAL_MEASURES = {
   STOP_FLOOR_REQUEST_FAILURE: 'js_sdk_stop_floor_request_failures',
   UPDATE_MEDIA_FAILURE: 'js_sdk_update_media_failures',
   UNMUTE_AUDIO_FAILURE: 'js_sdk_unmute_audio_failures',
-  UNMUTE_VIDEO_FAILURE: 'js_sdk_unmute_video_failures'
+  UNMUTE_VIDEO_FAILURE: 'js_sdk_unmute_video_failures',
+
+  ROAP_ANSWER_FAILURE: 'js_sdk_roap_answer_failure',
+  ROAP_GLARE_CONDITION: 'js_sdk_roap_glar_condition',
+
+  PEERCONNECTION_FAILURE: 'js_sdk_peerConnection_failure'
```

Comment:
Probably shouldn't have whitespaces unless there were supposed to be comments separating ROAP from the others

```suggestion
  UNMUTE_VIDEO_FAILURE: 'js_sdk_unmute_video_failures',
  ROAP_ANSWER_FAILURE: 'js_sdk_roap_answer_failure',
  ROAP_GLARE_CONDITION: 'js_sdk_roap_glar_condition',
  PEERCONNECTION_FAILURE: 'js_sdk_peerConnection_failure'
```